Problem: create-target-array-in-the-given-order
Submission ID: 1695687983
Submission Time: 2025-07-12 14:53:04
============================================================

State: SUCCESS
Status: Accepted
Language: Python

Runtime: 0 ms
Memory: 12.60 MB

Test Cases: 45/45 passed

Runtime Percentile: 100%
Memory Percentile: 6.093199999999996%

============================================================
RAW SUBMISSION DATA
============================================================
{
  "status_code": 10,
  "lang": "python",
  "run_success": true,
  "status_runtime": "0 ms",
  "memory": 12596000,
  "display_runtime": "0",
  "question_id": "1505",
  "elapsed_time": 46,
  "compare_result": "111111111111111111111111111111111111111111111",
  "code_output": "",
  "std_output": "",
  "last_testcase": "",
  "expected_output": "",
  "task_finish_time": 1752357183288,
  "task_name": "judger.judgetask.Judge",
  "finished": true,
  "total_correct": 45,
  "total_testcases": 45,
  "runtime_percentile": 100,
  "status_memory": "12.6 MB",
  "memory_percentile": 6.093199999999996,
  "pretty_lang": "Python",
  "submission_id": "1695687983",
  "status_msg": "Accepted",
  "state": "SUCCESS"
}